libfwupdplugin/fu-device.h

```diff
@@ -458,6 +458,15 @@ typedef guint64 FuDeviceInternalFlags;
  */
 #define FU_DEVICE_INTERNAL_AUTO_PAUSE_POLLING (1ull << 24)
 
+/**
+ * FU_DEVICE_INTERNAL_FLAG_ONLY_WFR_REMOVE_DELAY:
```

Comment:
don't like this name, any better?

Reply:
`NO_DELAYED_REMOVAL`? `NO_REMOVAL_DELAY`? `ONLY_WAIT_FOR_REPLUG`? `STRICT_WAIT_FOR_REPLUG`?

Reply:
I like ONLY_WIAT_FOR_REPLUG

Reply:
Ack, fixed.